app/controllers/api/debates_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Api::DebatesController < Api::ApplicationController
+  before_action :require_current_debate_not_closed
```

Comment:
don't we need also `require_current_debate`?

Reply:
We discussed it